converter/interceptor.go

```diff
@@ -0,0 +1,215 @@
+// The MIT License
```

Comment:
Probably best named as `grpc_interceptor.go` to avoid overloading the term as we otherwise use it